### ISSUE TITLE
Full results on Cityscapes

### ISSUE THREAD

**Issue by XiaorongLi-95** [zhangchbin/RCIL#7]
Nice work!
Could you publish the scripts and the corresponding results on Cityscapes?
I failed to reproduce the experimental results reported in the paper. I set the batch size as 24. The initial learning rate is 0.02 for the first training step and 0.001 for the next continual learning steps. I train the model for each step with 50 epochs as the paper suggested.

**Comment by zhangchbin** [comment]
Hi, @XiaorongLi-95 
We have added scripts for Cityscapes.

**Comment by XiaorongLi-95** [comment]
Thanks for your quick reply!
Could you provide detailed information about the environment and GPU resources?
Is your environment same as PLOP, CVPR 2021?
I installed libraries as in requirements of PLOP and used both four V100 and four 3090. But both of them obtain 1 lower final mIoU than the paper reported.
BTW, the line 17 in RCIL_cityscapes_1-1.sh should be "TASK=1-1" not "TASK=11-1".

**Comment by zhangchbin** [comment]
@XiaorongLi-95 
We usually use pytorch 1.3.1 on 4 2080ti gpus.
For the difference on cityscapes, I have fixed the initialization for Cityscapes dataset, and you can refer to [link](https://github.com/zhangchbin/RCIL/blob/main/segmentation_module.py#L239). If you have any results, please let me know. Thanks.

**Comment by XiaorongLi-95** [comment]
Thanks!
I have used your fixed initialization and used four v100 GPUs. The environment is based on PyTorch 1.8.1, Cuda 10.2, apex=0.1, inplace-abn=1.1.0. I obtained 63.6 final mIoU on Cityscapes 11-5(3 steps).

**Comment by zhangchbin** [comment]
hi, @XiaorongLi-95 
We found that high version pytorch can not reach the supposed performance due to the bug caused by InplaceABNSync, which has been fixed in our latest commitment. You can refer to our latest news. Sorry for the inconvenience caused.
